src/Codec/Compression/LZ4/Conduit.hsc

```diff
@@ -226,8 +235,8 @@ lz4fCreateCompressonContext = do
   -- that time would make us leak memory).
   addForeignPtrFinalizer haskell_lz4_freeCompressionContext ctxForeignPtr
   -- TODO The whole idea above is to avoid `mask`.
-  --      But we should check if `addForeignPtrFinalizer` itself is actually
-  --      async exception safe; if not, this is pointless.
```

Comment:
I'd like to keep TODO indents, as they make it obvious syntactically what is part of the TODO that should be removed when the TODO is done, and what is normal code commentary.